build.gradle.kts

```diff
@@ -1,5 +1,9 @@
+import korlibs.korge.gradle.configureAutoVersions
+
 plugins {
     alias(libs.plugins.korge) apply false
 }
 
+configureAutoVersions()
```

Comment:
I mean this here with my question. Sorry, was on mobile...

Reply:
https://github.com/korlibs/korge/blob/656078f3a41505f1b1625f39210edb3cb2110117/buildSrc/src/main/kotlin/korlibs/korge/gradle/KorgeGradlePlugin.kt#L120

Since the plugin is not applied in the root project, this is required.
That's because the `kproject` modules I have created doesn't specify a KorGE version, but uses the one available. Since there is no version available in the root, that code configures the version used:

https://github.com/korlibs/korge-jitto/blob/53fa5eea9b648ca1fd1b008121d26bd355474945/korge-jitto/kproject.yml#L2